metastats-domain/src/main/groovy/life/qbic/metastats/datamodel/MetaStatsPackageEntry.groovy

```diff
@@ -0,0 +1,11 @@
+package life.qbic.metastats.datamodel
+
+class MetaStatsPackageEntry {
+    String entryId
+    HashMap properties
```

Comment:
At least the id should not change. Make it final.

Reply:
0101820